content/contents+de.lr

```diff
@@ -1,76 +1,80 @@
 ---
 intro_body_2:
 
-Alle Ergebnisse stehen jedem zur Nutzung und Wiederverwendung frei.
+Alle Outputs sind offen für die Nutzung und Wiederverwendung durch jede(n).
 ---
 year_intro:
 
-Für den Open Data Day 2019 wollen wir noch einen drauf setzen. Der Fokus wird dieses auf vier Schwerpunkten liegen, auf die Open Data unserer Meinung nach positive Auswirkungen haben kann.
+Beim Open Data Day 2020 wollen wir die Community weiter wachsen lassen. Wir werden Minifinanzierungen für Schlüsselbereiche vergeben, von denen wir glauben, dass sie mit offenen Daten gelöst werden können. Die diesjährigen Tracks findest du unten.
 ---
-year_label_1: Offene **Forschungsdaten**
+year_label_1: Offene **Umweltdaten**
 ---
-year_label_2: Offene **Haushalte**
+year_label_2: Verfolgung der öffentlichen **Geldflüsse**
 ---
-year_label_3: Offene Daten für den **Umweltschutz**
+year_label_3: Offene **Kartographie**
 ---
-year_label_4: Offene Daten für **Menschenrechte**
+year_label_4: Offene Daten für **Gleichstellung**
 ---
 register_text: Veranstaltung hier eintragen
 ---
-events_registered: eingetragene Veranstaltungen
+events_registered: Bisher registrierte Events
 ---
 who_heading: An wen richtet sich der Open Data Day? An jede(n)!
 ---
 who_body:
 
-Wenn du eine Idee zur Verwendung offener Daten hast, egal ob bereits umgesetzt oder noch im Planungsstadium, etwas zur Verwendung offener Daten lernen möchtest oder einfach sehen willst, was am Open Data Day passiert, dann komm einfach bei einem der Events vorbei! Jeder ist herzlich willkommen an den Veranstaltungen teilzunehmen und sich einzubringen. Ganz egal welche Interessen dich mit offenen Daten verbinden oder wie dein aktueller Wissenstand zu offenen Daten ist: Organisatoren von Events rund um den Open Data Day sind bemüht ein interessantes Programm für jede(n) zu bieten!
+Wenn du eine Idee zur Verwendung offener Daten hast, egal ob bereits umgesetzt oder noch im Planungsstadium, etwas zur Verwendung offener Daten lernen möchtest oder einfach bei einem der Open Data Events in Österreich dabei sein möchtest, dann ist der Open Day ganz sicher etwas für dich! Trag ganz einfach deine Idee [hier online](https://docs.google.com/spreadsheets/d/17bnNQshA1gJw0UUa1KBZSCP34NFw4541exaqv2843N0/edit#gid=0) ein, sieh dir die [Liste der bereits organisierten Events](https://fusiontables.google.com/embedviz?viz=CARD&q=select+*+from+1moN6-tpBP5F2gsNkQqryVhoJP3_iZA3qVzAG6LPO&tmplt=3&cpr=2) an oder finde ein [Event auf der Karte](https://fusiontables.google.com/embedviz?q=select+col3+from+1moN6-tpBP5F2gsNkQqryVhoJP3_iZA3qVzAG6LPO&viz=MAP&h=false&lat=47.6924900372039&lng=14.71441445000005&t=1&z=8&l=col3&y=2&tmplt=2&hml=ONE_COL_LAT_LNG). Ganz egal welche Interessen dich mit offenen Daten verbinden oder wie dein aktueller Wissenstand zu offenen Daten ist: Organisatoren von Events rund um den Open Data Day sind bemüht ein interessantes Programm für jede(n) zu bieten!
```

Comment:
If we're merging the Austrian translation with the German translation, we should probably remove "in Österreich"